hyperdash/monitor.py

```diff
@@ -10,33 +10,37 @@
 from .code_runner import CodeRunner
 from .hyper_dash import HyperDash
 from .io_buffer import IOBuffer
-from .sdk_message import create_log_message
 from .server_manager import ServerManagerHTTP
 
+def get_logger(model_name, current_sdk_run_uuid, stdout_buffer):
```

Comment:
I like that we're sharing this, but it should probably be in a util or something. Its weird to me that the experiment imports monitor just for this func

Reply:
Agreed